src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -311,6 +325,33 @@ public List<DarCollectionSummary> getSummariesForRoleName(User user, String user
     return summaries;
   }
 
+  public DarCollectionSummary draftDarCollection(DarCollection sourceCollection) {
```

Comment:
nit: naming. I think this should more completely convey what it is doing. It's moving a collection from non-draft to draft, so something like `updateCollectionToDraft` or something similar.